x-pack/legacy/plugins/monitoring/public/components/elasticsearch/shard_allocation/_shard_allocation.scss

```diff
@@ -14,52 +14,57 @@ monitoring-shard-allocation {
   cluster-view {
     display: block;
   }
-  .parent {
-    padding-top: 14px;
-    border-left: 3px solid $euiColorSuccess !important;
-    &.red {
-      border-left: 3px solid $euiColorDanger !important;
-    }
-    &.yellow {
-      border-left: 3px solid $euiColorWarning !important;
-    }
-  }
-  td.unassigned {
+
+  .monUnassigned {
     vertical-align: middle;
     width: 150px;
   }
 
-  .child {
+  .monChildren {
+    padding-top: $euiSizeL;
+  }
+
+  .monChild {
     float: left;
     align-self: center;
+    background-color: $euiColorLightestShade;
+    margin: $euiSizeS;
+    border: 1px solid $euiColorMediumShade;
+    border-radius: $euiSizeXS;
+    padding: $euiSizeXS/2 0;
 
     &.index {
       border-left: $euiSizeXS solid $euiColorSuccess;
+
       &.red {
         border-left: $euiSizeXS solid $euiColorDanger;
       }
+
       &.yellow {
         border-left: $euiSizeXS solid $euiColorWarning;
       }
     }
-    background-color: $euiColorDarkShade;
-    margin: 5px;
-    .title {
-      padding: 5px 7px;
-      display: inline-block;
+
+    .monTitle {
+      padding: $euiSizeXS $euiSizeS;
       text-align: center;
-      font-size: 12px;
-      font: 10px sans-serif;
+      font-size: $euiSizeM;
       color: $euiColorGhost;
+      display: flex;
+      flex-direction: row;
+      align-items: center;
+
       a {
-        color: $euiColorGhost;
         text-decoration: none;
       }
+
+      .euiIcon,
```

Comment:
Is this needed? Gonna guess it's an icon next to some text? Likely a flexgroup can provide the spacing.